augur/data/schema-export-v2.json

```diff
@@ -188,6 +188,33 @@
                                         {"type": "string", "description": "color hex value", "pattern": "^#[0-9A-Fa-f]{6}$"}
                                     ]
                                 }
+                            },
+                            "legend": {
+                                "description": "Specify the entries displayed in the legend. This can be used to restrict the entries in the legend for display without otherwise affecting the data viz",
+                                "type": "array",
+                                "items": {
+                                    "type": "object",
+                                    "required": ["value"],
+                                    "properties": {
+                                        "value": {
+                                            "description": "value to associate with this legend entry. Used to determine colour. For non-continuous scales this also determines the matching between legend items and data.",
+                                            "type": ["string", "number"],
+                                            "$comment": "Continuous scales must use a numeric value. Other scales can use either."
+                                        },
+                                        "display": {
+                                            "description": "Label to display in the legend. Optional - `value` will be used if this is not provided.",
+                                            "type": ["string", "number"]
+                                        },
+                                        "bounds": {
+                                            "description": "(for continuous scales only) provide the lower & upper bounds to match data to this legend entry. Bounds from different legend entries must not overlap. Matching is (a, b] - exclusive of the lower bound, inclusive of the upper.",
```

Comment:
This description says:

> Bounds from different legend entries must not overlap.

But in the `_add_legend` function in `export_v2.py`, a comment says:

```
# NOTE: structure already validated by schema
```

Should we note somewhere that we do not actually check for overlapping legend entries or should we add logic to the `_add_legend` function to warn the user when the bounds overlap? We seem to provide warnings for other similar validation issues like requesting a color scale for values that do not exist in the tree.